src/Cms/AppErrors.php

```diff
@@ -190,8 +190,19 @@ protected function setWhoopsHandler(callable|HandlerInterface $handler): void
 	protected function getAdditionalWhoopsHandler(): CallbackHandler
 	{
 		return new CallbackHandler(function ($exception, $inspector, $run) {
-			$this->trigger('system.exception', compact('exception'));
-			error_log($exception);
+			$isLogged = true;
+
+			// allow hook to modify whether the exception should be logged
+			$isLogged = $this->apply(
+				'system.exception',
+				compact('exception', 'isLogged'),
+				'isLogged'
+			);
+
+			if ($isLogged === true) {
```

Comment:
Would it be better to turn this around and check for false here? It would be a bit more explicit to surpress warnings that way and I think it would also not lead to a breaking change.